packages/twilio/src/verification/index.ts

```diff
@@ -0,0 +1,9 @@
+import { TwilioVerification } from './TwilioVerification';
+
+export const useTwilioVerification = (
+  accountSid = process.env.TWILIO_ACCOUNT_SID,
+  authToken = process.env.TWILIO_AUTH_TOKEN,
+  verificationServiceSid = process.env.TWILIO_VERIFICATION_SERVICE_SID
+): TwilioVerification => new TwilioVerification({ accountSid, authToken }, verificationServiceSid);
+
+export { TwilioVerification };
```

Comment:
How about renaming it to `createTwilioVerification` to avoid confusion with React hooks?

Reply:
we are using the same _naming convention_ everywhere